utilities/blob_db/blob_db_impl.cc

```diff
@@ -1295,8 +1295,9 @@ std::pair<bool, int64_t> BlobDBImpl::SanityCheck(bool aborted) {
 Status BlobDBImpl::CloseBlobFile(std::shared_ptr<BlobFile> bfile) {
   assert(bfile != nullptr);
   Status s;
-  ROCKS_LOG_INFO(db_options_.info_log, "Close blob file %" PRIu64,
-                 bfile->BlobFileNumber());
+  ROCKS_LOG_INFO(db_options_.info_log,
+                 "Closing blob file %" PRIu64 ". Path: %s",
+                 bfile->BlobFileNumber(), bfile->PathName().c_str());
```

Comment:
Just file number is enough?

Reply:
Well, I added the path just because "New blob file created: " log line was adding the path, as well as the log line that I deleted from blob_file.cc. So wanted the "closing blob file" also to be similar to them in format. Also, I didn't want to delete some of the information that we were already printing. 

I don't have a strong preference either way. I'll let the current format stay for now, and may be remove it later if I don't find it useful.